tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -192,7 +192,7 @@ public MyTestAttribute(int value)
             Assert.Equal("int value", signature.Parameters.ElementAt(0).Label);
         }
         [Fact]
-        public async Task SignatureHelpforAttCtorMultipleArg()
+        public async Task SignatureHelpforAttCtorMultipleParam()
         {
             const string source =
 @"using System;
```

Comment:
Does this pass? It doesn't look like there's a "$$" anywhere in the markup?

Reply:
My bad, I see it. Anyway, it seems like the test is called "MultipleParam", but you're only testing in the first parameter position.